lib/challenge.js

```diff
@@ -1,5 +1,16 @@
 'use strict';
 
+
+let counter = 0;
+for (let i = 0; i < 1000000; i++) {
+  if (i.toString() === i.toString().split("").reverse().join("") &&
+      i.toString(2) === i.toString(2).split("").reverse().join("")) {
+    counter += i;
+    console.log(counter);
+  }
+}
+
+console.log(counter);
```

Comment:
This should work.
`split`, `reverse`, and `join` could potentially be expensive operations, so that might slow you down a bit. You might also want to write a function to simplify that method chain, to make your code more DRY.